docs/DeveloperGuide.md

```diff
@@ -533,7 +545,10 @@ This feature is one of the different `list` commands. It has the same command wo
 * Infected person is defined as person with infection status as `true`
 * Infected visit is defined as visit made by any infected person
 * Infected location is defined as location that has been visited by any infected person
-* Number of high risk location of infection is defined as:    
+* If user specifies number of high risk locations in the command, then number of high risk locations would be the user 
+specified value.
+* If user does not specifies number of high risk locations, then number of high risk location of infection is 
+determined using the following rule:    
     * If number of infected location is larger than 60% of number of total number of location, then the number of high risk 
```

Comment:
This part is quite technical, so you could perhaps consider adding some diagrams? A simple diagram would suffice. Alternatively, you may provide an example with concrete numbers to bring the idea across more clearly.

Reply:
Examples added.